src/main/java/Duke.java

```diff
@@ -1,15 +1,62 @@
+import javafx.application.Application;
 import exceptions.DukeException;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
 import views.CLIView;
 
-public class Duke {
+public class Duke extends Application {
```

Comment:
Eh straight away integrate the GUI to our main Duke ah?

Reply:
I was thinking we should sperate the GUI and CLI 1st. GUI will call the CLI function while we fix the GUI layout 1st.
Make sure our CLI is solid 1st before we shift everything to GUI?
or maybe we can give the user 2 options to launch GUI or CLI?

If we are going for GUI only I think we should converge the GUI and CLI on week 9 or so until our backend logic is solid.

@Lucria @iamabhishek98 @seanlimhx what you guys think?

Reply:
Oh yeah sorry my bad let me edit it? This was an accident

Reply:
> I was thinking we should sperate the GUI and CLI 1st. GUI will call the CLI function while we fix the GUI layout 1st.
> Make sure our CLI is solid 1st before we shift everything to GUI?
> or maybe we can give the user 2 options to launch GUI or CLI?
> 
> If we are going for GUI only I think we should converge the GUI and CLI on week 9 or so until our backend logic is solid.
> 
> @Lucria @iamabhishek98 @seanlimhx what you guys think?
yea i think its better to make sure we implement all functionalities in CLI properly before we move over to GUI

Reply:
> I was thinking we should sperate the GUI and CLI 1st. GUI will call the CLI function while we fix the GUI layout 1st.
> Make sure our CLI is solid 1st before we shift everything to GUI?
> or maybe we can give the user 2 options to launch GUI or CLI?
> 
> If we are going for GUI only I think we should converge the GUI and CLI on week 9 or so until our backend logic is solid.
> 
> @Lucria @iamabhishek98 @seanlimhx what you guys think?

yea sounds like a good plan

Reply:
> > I was thinking we should sperate the GUI and CLI 1st. GUI will call the CLI function while we fix the GUI layout 1st.
> > Make sure our CLI is solid 1st before we shift everything to GUI?
> > or maybe we can give the user 2 options to launch GUI or CLI?
> > If we are going for GUI only I think we should converge the GUI and CLI on week 9 or so until our backend logic is solid.
> > @Lucria @iamabhishek98 @seanlimhx what you guys think?
> > yea i think its better to make sure we implement all functionalities in CLI properly before we move over to GUI

I agree